### DISCUSSION TITLE
Queries regarding Enterprise Cloud plan

### DISCUSSION THREAD

**Discussion by himankgupta81** [community/community#23704] [Enterprise]
<p>A couple of questions regarding the GitHub Enterprise plan for cloud:<br>
1- Can I have multiple organizations for one Enterprise cloud plan similar to the way we do it for GitHub Enterprise server?<br>
2- How can we Suspend a user account in Enterprise cloud plan. Again, we can do it for the Enterprise Server plan but not sure how to do it in Enterprise Cloud.</p>

**Comment by byrneh** [comment, marked as answer]
<p><a class="mention" href="/himankgupta81">@himankgupta81</a>,</p>
<ol>
<li>Yes enterprise plan supports multiple organizations, similar to what you have for GHES</li>
<li>You don’t own manage an actual users personal GitHub account, so the same concept of suspending that account does not exists for GHEC (Cloud).<br>
You will need to remove a users permissions on all enterprise resources (organizations).</li>
</ol>
